### ISSUE TITLE
[NPC] Calamoth Ashbread 41522

### ISSUE THREAD

**Issue by sarahkay88** [WoWManiaUK/Blackwing-Lair#331]
QUEST - https://www.wowhead.com/quest=26128/the-battle-of-thandol-span
NPC -https://www.wowhead.com/npc=42160/thargas-anvilmar
this is a 6 quest chain starting at quest  https://www.wowhead.com/quest=26327/anvilmar-the-hero
For this quest you are to ride to the bridge and defeat NPC: https://www.wowhead.com/npc=41522/calamoth-ashbeard however he is not there so cannot be defeated. this prevents you from completing this quest and also obtaining and completing 3 other quest which are
quest : https://www.wowhead.com/quest=26139/into-arathi
quest: https://www.wowhead.com/quest=26093/northfold-manor
quest: https://www.wowhead.com/quest=26095/stromgarde-badges

**Comment by sarahkay88** [comment]
after further reading it appears https://www.wowhead.com/quest=26139/into-arathi can be picked up from the heros board. I will check to see if these can be completed that way

**Comment by sarahkay88** [comment]
![world of warcraft 01_01_2019 22_13_45](https://user-images.githubusercontent.com/42302580/50576848-b9263580-0e12-11e9-85ad-dc19120e0faf.png)
Unable to check the hero board in ironfordge does not display this quest for me

**Comment by Echinaceax** [comment]
The quest iton is silver meaning you are to low lvl fot it atm ...test again when you are the right lvl for it 

i just had to level a few more to be able to take the horde one 

(just a thought ofc)

**Comment by sarahkay88** [comment]
the quest area is below the level I am at. the area to lv is 25-30 I am lvl 31. I was able to obtain the one to hinterlands

**Comment by Echinaceax** [comment]
hmm...
Ok, just same quest but different faction, thats how it was for me (available at 24)

**Comment by sarahkay88** [comment]
yea when levelling the mage the hero boards displayed all the hero quest be these higher or lower n you could take which ever one you wanted. but not in ironfordge

**Comment by Echinaceax** [comment]
ah gotcha

**Comment by Echinaceax** [comment]
"um that's 2 issues in one report, should a separate issue be made?"

I would think yes
i decided to make 2 reports with the Helcular's Rod / Helcular's Rod quest

**Comment by Dranah** [comment]
Ok so this is supposed to be some kind of event as it looks and Calamoth spawns during it.
Need to be scripted in order for NPC to spawn.

**Comment by Dranah** [comment]
https://www.wowhead.com/quest=28573/heros-call-arathi-highlands is available from the board only after completing this quest, as it is a prerequisite for it.
